env/includes/3dsmax/project.yml

```diff
@@ -13,76 +13,21 @@
 # is running in its project level configuration
 
 includes:
-- ../common/frameworks.yml
```

Comment:
I've done a second pass per your request @manneohrstrom to share tk-multi-shotgunpanel config in the apps.yml include